doc/source/structures/misc/stack.rst

```diff
@@ -0,0 +1,126 @@
+.. _stack:
+
+Stack
+====
+
+A :struct:`Stack` is a collection of any type in kOS. Stacks work according to `Last-In first-out <https://en.wikipedia.org/wiki/FIFO_and_LIFO_accounting>`_ principle. It may be useful to contrast
+:struct:`Stack` with :struct:`stack` to better understand how both structures work. You can read more about stacks on `Wikipedia <https://en.wikipedia.org/wiki/Stack_(abstract_data_type)>`_.
```

Comment:
>It may be useful to contrast  `Stack` with `stack` to better understand how both structures work 

I assume you meant `Queue` for the second link there?